### PR TITLE
docs: documented very_good test argument terminator

### PR DESCRIPTION
**Pull request by alestiago**
## Description

Documents the usage of `--` to specify flutter specific arguements.

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [X] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: site/docs/commands/test.md [hunk 1]
```diff
@@ -31,6 +31,14 @@ very_good test [arguments]
 Run "very_good help" to see global options.
 ```
 
+### Passing Flutter specific arguements
```
Comment (Mayaa-s) [inline site/docs/commands/test.md]: "arguments" is misspelled

File: site/docs/commands/test.md [hunk 2]
```diff
@@ -31,6 +31,14 @@ very_good test [arguments]
...
+
+The `flutter test` command exposes more arguements than those available through `very_good test`. Despite this, you can use the arguement terminator `--` to signify the end of `very_good test` command options and the beginning of `flutter test` command options.
```
Comment (Mayaa-s) [inline site/docs/commands/test.md]: argument is misspelled here too. should read "The flutter test command exposes more arguments than those available through very_good test. Despite this, you can use the argument terminator -- to signify the end of very_good test command options and the beginning of flutter test command options"

File: site/docs/commands/test.md [hunk 3]
```diff
@@ -31,6 +31,14 @@ very_good test [arguments]
 Run "very_good help" to see global options.
 ```
 
+### Passing Flutter specific arguments
+
+The `flutter test` command exposes more arguments than those available through `very_good test`. Despite this, you can use the argument terminator `--` to signify the end of `very_good test` command options and the beginning of `flutter test` command options.
+
+For example, if you wish to run `flutter test --no-track-widget-creation` you can simply do `very_good test -- --no-track-widget-creation`.
+
+You may also target specific files by using the argument terminator; for example `very_good test -- test/very_good_test/**`. However, when targeting a subset of the project tests optimization will always be disabled.
```
Comment (Mayaa-s) [inline site/docs/commands/test.md:40]: Small grammatical changes here (commas, and changed "you can simply do" to "you can simply use" here's the revised version:

For example, if you wish to run flutter test --no-track-widget-creation, you can simply use very_good test -- --no-track-widget-creation.

You may also target specific files by using the argument terminator; for example, very_good test -- test/very_good_test/**. However, when targeting a subset of the project's tests, optimization will always be disabled.

Comment (Mayaa-s) [inline site/docs/commands/test.md:40]:
```suggestion
For example, if you wish to run `flutter test --no-track-widget-creation` you can simply use `very_good test -- --no-track-widget-creation`.

You may also target specific files by using the argument terminator; for example, `very_good test -- test/very_good_test/**`. However, when targeting a subset of the project tests, optimization will always be disabled.
```

**Review by Mayaa-s — COMMENTED** [review]
Minor changes, looks good otherwise.

**Review by erickzanardo — DISMISSED** [review]

File: site/docs/commands/test.md [hunk 4]
```diff
@@ -31,6 +31,14 @@ very_good test [arguments]
 Run "very_good help" to see global options.
 ```
 
+### Passing Flutter specific arguments
+
+The `flutter test` command exposes more arguments than those available through `very_good test`. Despite this, you can use the argument terminator `--` to signify the end of `very_good test` command options and the beginning of `flutter test` command options.
```
Comment (alestiago) [inline site/docs/commands/test.md:36]:
```suggestion
The `flutter test` command exposes more arguments than those available through `very_good test`. Despite this, you can use the argument terminator `--` to signify the end of `very_good test` command options and the beginning of `flutter test` command options. Making all `flutter test` arguments available!
```

**Review by omartinma — APPROVED** [review]
Awesome!

**Review by erickzanardo — APPROVED** [review]

File: site/docs/commands/test.md [hunk 5]
```diff
@@ -31,6 +31,14 @@ very_good test [arguments]
 Run "very_good help" to see global options.
 ```
 
+### Passing Flutter specific arguments
+
+The `flutter test` command exposes more arguments than those available through `very_good test`. Despite this, you can use the argument terminator `--` to signify the end of `very_good test` command options and the beginning of `flutter test` command options; making all `flutter test` arguments available!
+
+For example, if you wish to run `flutter test --no-track-widget-creation` you can simply use `very_good test -- --no-track-widget-creation`.
+
+You may also target specific files by using the argument terminator; for example, `very_good test -- test/very_good_test/**`. However, when targeting a subset of the project tests, optimization will always be disabled.
```
Comment (renancaraujo) [inline site/docs/commands/test.md:40]: The usage for that case is `very_good test test/very_good_test/**``

Comment (alestiago) [inline site/docs/commands/test.md:40]:
```suggestion
```

File: site/docs/commands/test.md [hunk 6]
```diff
@@ -31,6 +31,13 @@ very_good test [arguments]
 Run "very_good help" to see global options.
 ```
 
+### Passing Flutter specific arguments
+
+The `flutter test` command exposes more arguments than those available through `very_good test`. Despite this, you can use the argument terminator `--` to signify the end of `very_good test` command options and the beginning of `flutter test` command options; making all `flutter test` arguments available!
+
+For example, if you wish to run `flutter test --no-track-widget-creation` you can simply use `very_good test -- --no-track-widget-creation`.
+
+
```
Comment (alestiago) [inline site/docs/commands/test.md:40]:
```suggestion
```

**Review by renancaraujo — APPROVED** [review]

**Pull request merged**